stanford_news.module

```diff
@@ -6,28 +6,28 @@
  */
 
 use Drupal\Component\Utility\Html;
+use Drupal\views\ViewExecutable;
+use Drupal\node\NodeInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Component\Utility\UrlHelper;
 
 /**
  * Implements hook_theme().
  */
 function stanford_news_theme() {
   return [
-    'node__stanford_news__full' => [
-      'template' => 'node--stanford-news',
-      'base hook' => 'node'
-    ],
     'field__node__su_news_topics' => [
-      'template' => 'field--node--su-news-topics',
+      'template' => 'field/field--node--su-news-topics',
       'base hook' => 'field'
```

Comment:
@boznik can you confirm that this field template is still needed. There is a todo in the template referencing an issue that was merged in 2018.